ccd-definition/ComplexTypes/EmployerDetailsLRspec.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "ID": "AdmitPartEmployer",
+    "ListElementCode": "employerDetailsLabel",
+    "FieldType": "Label",
+    "ElementLabel": "## Name of employer (if employed by someone else) \n",
```

Comment:
In the miro board I can't see the ## before the label. Are them special ccd characters?

Reply:
This is used for the style of the heading to make the font to be bold and larger